app/src/main/java/appricottsoftware/clarity/sync/ClarityClient.java

```diff
@@ -148,4 +153,62 @@ public void metadataUpVoteRequest(Context context, JsonHttpResponseHandler handl
             e.printStackTrace();
         }
     }
+
+
+    public void createChannel(int uid, String name, String imageURL, Context context, JsonHttpResponseHandler handler) {
+        // Create the rest client and add header(s)
+        //pass in metadata
+
+        AsyncHttpClient client = new AsyncHttpClient();
+
+        JSONObject jsonParams = new JSONObject();
+
+        JSONObject aMetadata = new JSONObject();
+        JSONObject bMetadata = new JSONObject();
+
+        try {
+            aMetadata.put("genre", "politics");
```

Comment:
Since these parameters are hard coded, maybe consider adding a TODO here to eventually take in parameters.

Reply:
TODO added.